recipes/wt/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,5 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

```suggestion
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
```

Due the follow error:

```
/Users/uilian/.conan/data/wt/4.9.1/_/_/package/79368ad163a66be578300e20c99b158838f08f91/include/Wt/WGlobal.h:25:55: error: no member named 'make_unique' in namespace 'std'
    auto make_unique(Args&& ...args) -> decltype(std::make_unique<T, Args...>(std::forward<Args>(args)...))

```

---

@uilianries 
Thank you for your review!
Wt began to required C++14 since 4.5.0.
(https://github.com/emweb/wt/releases/tag/4.5.0)
I have to write if-else expression.

I will try.

---

Fixed